feature-libs/storefinder/assets/translations/en/store-finder.ts

```diff
@@ -2,6 +2,7 @@ export const storeFinder = {
   storeFinder: {
     openUntil: 'Open until',
     closed: 'Closed',
+    clickBack: 'Back',
```

Comment:
Can we change this translation key to `back`? I think will be more general and consistent.